pkg/secrets/pushtofile/secret_group.go

```diff
@@ -217,6 +235,25 @@ func newSecretGroup(annotations map[string]string, groupName string) (*SecretGro
 		}
 	}
 
+	// First-pass at provided template rendering with dummy secret values
+	// This first-pass is limited for templates that branch conditionally on secret values
+	// Relying logically on specific secret values should be avoided
+	if len(fileTemplate) > 0 {
+		dummySecrets := []*Secret{}
+		for _, secretSpec := range secretSpecs {
+			dummySecrets = append(dummySecrets, &Secret{Alias: secretSpec.Alias, Value: "REDACTED"})
+		}
+
+		err = pushToWriter(ioutil.Discard, groupName, fileTemplate, dummySecrets)
+		if err != nil {
+			return nil, []error{fmt.Errorf(
+				`file template for secret group %q cannot be used as written: %s`,
```

Comment:
I think let's change
```
file template for secret group %q cannot be used as written: %s
```
to
```
unable to use file template for secret group %q: %s
```

to
1. Front-load the verb
2. I think the `as written` might be redundant